libs/internal/PeMaths/CPeTransform.h

```diff
@@ -26,14 +26,14 @@ namespace engine {
 			CPeTransform(
 				const CPeVector3& p_position,
 				const CPeQuaternion& p_orientation,
-				const CPeMatrix4& p_transformMatrix,
 				const CPeVector3& p_size
 			) :
 				m_position(p_position),
 				m_orientation(p_orientation),
-				m_transformMatrix(p_transformMatrix),
+				m_transformMatrix(1,0,0,0,1,0,0,0,1,0,0,0),
```

Comment:
Ok why not, on aurait pu juste rajouter un constructeur mais j'entend

Reply:
c'est surtout que ça sert vraiment a rien de donner la matrice de transformation vu que c'est du read-only. si on veut rajouter un constructeur, ça sera la matrice toute seule, et faudra calculer tous les autres membres à partir de ça.